api/src/helpers/transferCredits.ts

```diff
@@ -186,36 +189,30 @@ const handleApCalcAB = (
     return true;
   } else if (transfer.count == 1) {
     // Not okay because we don't know whether this is AB or the subscore
-    // console.log(
-    //   `x FAILED:  x${transfer.count}    '${transferName}'      has ambiguity: AP Calculus AB vs the BC Subscore`,
-    // );
     return false;
   } else {
     // The user has AP Calculus AB 2+ times and no AB subscore,
     // so we assume one of them is the BC subscore and the other isn't
     // Insert the BC subscore here, then proceed normally so that AB is also inserted
-    // console.log(`+ MATCHED:       '${transferName}' with the AP Calc AB Subscore on the BC test AS WELL AS:`);
     toInsertAp.push({
       examName: AP_CALC_AB_SUBSCORE,
-      score: null,
+      score: transfer.score ?? null,
       units: 0, // The units will be counted for the actual AB later, and the subscore will remain 0 units
     });
     return true;
   }
 };
 
 /** Handle organizing an AP exam, returning whether it was successfully categorized */
-export const organizeApExam = (
-  transfer: TransferredMiscSelectedRow,
-  transferName: string,
+const organizeApExam = (
+  transfer: GroupedUncategorizedTransfer,
   toInsertAp: TransferredApExamRow[],
   hasSubscore: boolean,
   allAps: ApExamBasicInfo[],
 ): boolean => {
-  const bestMatch = tryMatchAp(transferName, allAps);
+  const bestMatch = tryMatchAp(transfer.courseName, allAps);
   if (!bestMatch) {
     // Could not match; leave it here
```

Comment:
"Leave it here" is a reference to the misc table, which is not necessary in this case.

```suggestion
  if (!bestMatch) return false;
```

Reply:
Fixed various comments in this file